src/hotspot/share/runtime/arguments.cpp

```diff
@@ -1487,6 +1490,12 @@ void Arguments::set_use_compressed_klass_ptrs() {
 #endif // _LP64
 }
 
+static void validate_ergonomics_profile() {
+  if (strcmp(ErgonomicsProfile, "shared") != 0 && strcmp(ErgonomicsProfile, "dedicated") != 0) {
```

Comment:
I've seen strcmp(ErgonomicsProfile, "shared") a few times in the code - extract to a common method?

Reply:
Not sure there is value in that at this stage.